src/test/java/training/article/responsesuccessfully.json

```diff
@@ -0,0 +1,19 @@
+{
+  "article": {
+    "title": "#string",
+    "slug": "#string",
+    "body": "#string",
+    "createdAt": "#string",
+    "updatedAt": "#string",
+    "description": "#string",
+    "tagList": [],
+    "author": {
+      "username": "#string",
+      "bio": "##string",
+      "image": "#string",
+      "following": "#boolean"
+    },
+    "favorited": "#boolean",
+    "favoritesCount": "#number"
+  }
+}
```

Comment:
check the name of the file. Do you think the name of the file define this content ?